### ISSUE TITLE
Support latest Raspbian Wheezy (May-2015)?

### ISSUE THREAD

**Issue by IvanExpert** [RasppleII/a2server#7]
A2SERVER and A2CLOUD were last tested on Raspbian Wheezy from Feb 2015. No idea if there are issues when installed on later version.

**Comment by knghtbrd** [comment]
I can say that I am MOSTLY up to date with current Raspbian Wheezy 
and A2SERVER and A2CLOUD both work.  I say mostly because I have not 
updated the following packages:

libraspberrypi-bin         1.20150820-1   1.20150923-1
libraspberrypi-dev         1.20150820-1   1.20150923-1
libraspberrypi-doc         1.20150820-1   1.20150923-1
libraspberrypi0            1.20150820-1   1.20150923-1
raspberrypi-bootloader     1.20150820-1   1.20150923-1

I was worried about breaking A2SERVER due to kernel module stuff, but 
it seems that Raspbian's kernel has included DDP support for some 
time since the stuff to compile it has been commented out of A2SERVER 
for awhile now.

Since jessie is not yet Raspbian's official release and there's a 
separate ticket for that, I'll close this upon verifying it works and 
adding current bootloader versions to the script to silence the 
warning.

Joseph

**Comment by IvanExpert** [comment]
Yeah, I requested they add the AppleTalk kernel module a while back. Then it broke severely in kernels 3.12 through 3.15 (in Linux generally, not just Raspbian), and so I filed a report on it, and it got fixed. I wouldn't expect any ordinary updates going forward to break it unless somebody breaks it again.

Jessie appears to be a Raspbian official release now, alongside Wheezy (which doesn't appear to be being updated): https://www.raspberrypi.org/downloads/raspbian/
